### PR TITLE
GridTile refactoring

### PR DESCRIPTION
**Pull request by gcofficial**

### CODE REVIEW

File: src/components/grid/Orders.vue [hunk 1]
```diff
@@ -126,15 +144,19 @@ export default {
...
   },
 };
 
 </script>
 
 <style lang='scss'>
-// @import 'perfect-scrollbar/dist/css/perfect-scrollbar';
+@import 'perfect-scrollbar/dist/css/perfect-scrollbar';
 @import 'variables';
+.gridTile__content--orders {
```
Comment (xerox-h20) [inline src/components/grid/Orders.vue]: Perhaps this should be removed

File: src/components/TheHeaderWidgetsDropdown.vue [hunk 2]
```diff
@@ -1,76 +1,47 @@
...
-              {title: 'Documents'},
-            ]),
+          items: this.getGridData.filter((item) => getWidgetType(item.name) === 'property'),
         },
         {
           name: 'views',
           items: [
+            ...defaultViews,
+            ...this.getSavedViews,
             {
```
Comment (xerox-h20) [inline src/components/TheHeaderWidgetsDropdown.vue]: The merge resulted in an extra item in the dropdown. See attachment.

File: src/components/grid/Orders.vue (hunk 1 above)
Comment (gcofficial) [reply src/components/grid/Orders.vue]: Done.

File: src/components/TheHeaderWidgetsDropdown.vue (hunk 2 above)
Comment (gcofficial) [reply src/components/TheHeaderWidgetsDropdown.vue]: Fixed

**Pull request merged**